ext/yarp/extension.h

```diff
@@ -3,6 +3,7 @@
 
 #include <ruby.h>
 #include <ruby/encoding.h>
+#include <ruby/internal/special_consts.h>
```

Comment:
What's this include do?

Reply:
It allows us to use RTEST to check if filepath is nil [here](https://github.com/Shopify/yarp/blob/f2345416942939637dafd2fcb8bf444f840219d6/ext/yarp/extension.c#L96-L99)